packages/schema-dot-org-json-ld-components/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "schema-dot-org-json-ld-components",
+  "version": "0.1.0",
+  "description": "Components to generate schema.org json-ld markup.",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha --compilers js:babel-core/register"
```

Comment:
consider running `jest` from the root, running things in each package is slow and repetitive